qemu/tests/qemu_img_convert_with_rate_limit.py

```diff
@@ -48,4 +50,9 @@ def run(test, params, env):
     logging.debug("Verify md5 value of the temporary file")
     img_utils.check_md5sum(guest_temp_file, md5sum_bin, session,
                            md5_value_to_check=md5_value)
+    logging.debug("Remove the converted img file")
+    try:
+        target.remove()
+    except Exception:
+        print('The source file did not be deleted!')
```

Comment:
Please refer to the comments above.
Besides, what about moving remove to the end, in case there is any failure, session.close will not be executed